Marlin/src/lcd/extui/lib/mks_ui/draw_error_message.h

```diff
@@ -25,6 +25,10 @@
 extern "C" { /* C-declarations for C++ */
 #endif
 
+#ifndef PGM_P
+  #define PGM_P const char *
+#endif
+
 extern void lv_draw_error_message(PGM_P const msg);
```

Comment:
`PGM_P` specifically refers to strings stored in Flash and defined with `PROGMEM` or `PSTR`. It should not be used for C strings residing in SRAM or other static strings with no `PROGMEM` specifier.

Reply:
I think this code isnt used anywhere... but the original MKS PR doesn't compile without that define. I will try test without this code.
I just didnt remove because the MKS can have a reason for that code...

Reply:
We'll have to pass it on to MKS. `PGM_P` is meant to be a clue about the string storage.

Reply:
@makerbase-mks the code above isn't used anywhere...I compiled and run a version without it with apparently no problem. Can I remove or comment it by now? It will have a future usage?